src/Home.jsx

```diff
@@ -1,14 +1,41 @@
-import React from "react";
+import React, { Component } from "react";
 import DisplayProductData from "./components/DisplayProductData";
-import { Header } from 'semantic-ui-react'
+import { Icon } from 'semantic-ui-react'
+import DisplayCategoriesList from './components/DisplayCategoriesList'
+import axios from 'axios'
+import { getData } from "./modules/productData";
+
+class Home extends Component {
+
+  onChangeHandler = async (event) => {
+    let selectedCategory = event.target.textContent.toLowerCase()
+    let response = await axios.get(
+      `/categories/${selectedCategory}`, 
+    ) 
```

Comment:
You need to update the state of the products that you now have in the state in your `DisplayProductData`, you need to move that state one level up and then change is with this method as well